examples/dgmulti_2d/elixir_mhd_weak_blast_wave.jl

```diff
@@ -0,0 +1,55 @@
+
+using OrdinaryDiffEq
+using Trixi
+
+###############################################################################
+# semidiscretization of the compressible ideal GLM-MHD equations
+
+equations = IdealGlmMhdEquations2D(1.4)
+
+initial_condition = initial_condition_weak_blast_wave
+
+surface_flux = (flux_lax_friedrichs, flux_nonconservative_powell)
+volume_flux = (flux_hindenlang_gassner, flux_nonconservative_powell)
+dg = DGMulti(polydeg=3, element_type = Quad(), approximation_type = Polynomial(),
+             surface_integral = SurfaceIntegralWeakForm(surface_flux),
+             volume_integral = VolumeIntegralFluxDifferencing(volume_flux))
+
+num_cells_per_dimension = 16
+vertex_coordinates, EToV = StartUpDG.uniform_mesh(dg.basis.elementType, num_cells_per_dimension)
+vertex_coordinates = map(x -> 2.0 .* x, vertex_coordinates) # map domain to [-2, 2]^2
+mesh = VertexMappedMesh(vertex_coordinates, EToV, dg, is_periodic=(true, true))
+
+semi = SemidiscretizationHyperbolic(mesh, equations, initial_condition, dg)
+
+###############################################################################
+# ODE solvers, callbacks etc.
+
+tspan = (0.0, 0.4)
+ode = semidiscretize(semi, tspan)
+
+summary_callback = SummaryCallback()
+
+analysis_interval = 100
+analysis_callback = AnalysisCallback(semi, interval=analysis_interval, uEltype=real(dg))
+alive_callback = AliveCallback(alive_interval=10, analysis_interval=analysis_interval)
+
+cfl = 1.0
+stepsize_callback = StepsizeCallback(cfl=cfl)
+
+glm_speed_callback = GlmSpeedCallback(glm_scale=1.0, cfl=cfl)
```

Comment:
To be honest, I'm not sure what it does. I can switch it back to `glm_scale=0.5`.

Reply:
This `glm_scale` is for timestep security. In theory, this scaling would always be 1. But, because the advective eigenvalues are only estimates, we put in this extra scaling to make sure that the GLM waves don't move "too fast" (that is faster than the fast magneto-acoustic waves in the Riemann fan). We typically, in an ad hoc way, choose `glm_scale=0.5`